test/flows/register_a_birth_flow_test.rb

```diff
@@ -0,0 +1,542 @@
+    setup do
+      testing_node :no_birth_certificate_result
+      add_responses country_of_birth?: "jordan",
+                    who_has_british_nationality?: "mother",
+                    married_couple_or_civil_partnership?: "no"
+    end
+
+    context "same_country" do
+      setup do
+        add_responses where_are_you_now?: "same_country"
+      end
+
+      should "render Afghanistan guidance if child born in Afghanistan" do
+        add_responses country_of_birth?: "afghanistan"
+        assert_rendered_outcome text: "It’s illegal in Afghanistan to have a child outside of marriage"
+      end
+
+      should "render Iran guidance if child born in Iran" do
+        add_responses country_of_birth?: "iran"
+        assert_rendered_outcome text: "It’s illegal in Iran to register the birth of a child if the parents aren’t married"
+      end
+
+      should "render Afghanistan guidance if child born in Iraq" do
+        add_responses country_of_birth?: "iraq"
+        assert_rendered_outcome text: "It’s illegal in Iraq to register the birth of a child if the parents aren’t married"
+      end
+
+      should "render Jordan guidance if child born in Jordan" do
+        add_responses country_of_birth?: "jordan"
+        assert_rendered_outcome text: "It’s illegal in Jordan to have a child outside of marriage"
+      end
+
+      should "render Kuwait guidance if child born in Kuwait" do
+        add_responses country_of_birth?: "kuwait"
+        assert_rendered_outcome text: "It’s illegal in Kuwait to have a child outside of marriage or within the first 6 months of marriage"
+      end
+
+      should "render Oman guidance if child born in Oman" do
+        add_responses country_of_birth?: "oman"
+        assert_rendered_outcome text: "It’s illegal in Oman to have a child outside of marriage"
+      end
+
+      should "render Pakistan guidance if child born in Pakistan" do
+        add_responses country_of_birth?: "pakistan"
+        assert_rendered_outcome text: "It’s illegal in Pakistan to have a child outside of marriage"
+      end
+
+      should "render Qatar guidance if child born in Qatar" do
+        add_responses country_of_birth?: "qatar"
+        assert_rendered_outcome text: "It’s illegal in Qatar to have a child outside of marriage"
+      end
+
+      should "render Saudi Arabia guidance if child born in Saudi Arabia" do
+        add_responses country_of_birth?: "saudi-arabia"
+        assert_rendered_outcome text: "It’s illegal in Saudi Arabia to have a child outside of marriage"
+      end
+
+      should "render United Arab Emirates guidance if child born in United Arab Emirates" do
+        add_responses country_of_birth?: "united-arab-emirates"
+        assert_rendered_outcome text: "It’s illegal in the United Arab Emirates to have a child outside of marriage"
+      end
+    end
+
+    should "render other country guidance if birth not registered in country of birth" do
```

Comment:
I looked back to marriage abroad and Kevin used this `random_country` [method](https://github.com/alphagov/smart-answers/blob/ba95009797fdf422936b81d8e8f9a478cc8523f4/test/flows/marriage_abroad_flow_test.rb#L73) or looped [through each country](https://github.com/alphagov/smart-answers/blob/ba95009797fdf422936b81d8e8f9a478cc8523f4/test/flows/marriage_abroad_flow_test.rb#L183) so wondering if we should do one of those here, to catch the unlikely event it disappears for one of the countries in question?

Reply:
I'll remove this commit and leave the loop in.

Reply:
👍